en/backup-restore-by-ebs-snapshot-faq.md

```diff
@@ -35,3 +35,11 @@ This document addresses common questions and solutions related to EBS snapshot b
 **Symptom:** Scheduled backup can't be completed in the expected window due to the cost of snapshot size calculation.
 
 **Solution:** By default, both full size and incremental size are calculated by calling the AWS service, which might take several minutes. You can set `spec.template.calcSizeLevel` to `full` to skip incremental size calculation, set it to `incremental` to skip full size calculation, and set it to `none` to skip both calculations.
+
+## How to configure the TTL for the backup init job
+
+Backup init job will do backup preparation work including pause GC, some PD schedulers and also suspend lightning.  Default TTL of 10 minutes is associated with the init job in case it could stuck.  You can change the TTL by setting `spec.template.volumeBackupInitJobMaxActiveSeconds` attribute of spec of volumebackup.
```

Comment:
```suggestion
The backup init job will handle backup preparations, including pausing GC, certain PD schedulers, and suspending Lightning. By default, a TTL of 10 minutes is associated with the init job in case it gets stuck. You can change the TTL by setting the `spec.template.volumeBackupInitJobMaxActiveSeconds` attribute of spec of volumebackup.
```